src/main/java/com/dmytrohuk/weborganizer/users/User.java

```diff
@@ -28,16 +21,24 @@ public class User {
     private String firstName;
 
     @Column(name = "last_name")
-    private String lastName;
+    private String surname;
 
     private String email;
 
-    public User(String username, String password, String firstName, String lastName, String email) {
+    private String address;
+
+    @Column(name = "role")
+    private String role;
```

Comment:
As you cannot have arbitrary string as a role, the best solution is to specify it as enum and then use `@Enumerated` to allow JPA to handle enum <-> string conversion (https://www.baeldung.com/jpa-persisting-enums-in-jpa)